management_api_app/api/routes/workspaces.py

```diff
@@ -23,12 +24,22 @@ async def retrieve_active_workspaces(workspace_repo: WorkspaceRepository = Depen
 
 @router.post("/workspaces", status_code=status.HTTP_202_ACCEPTED, response_model=ResourceInResponse, name=strings.API_CREATE_WORKSPACE)
 async def create_workspace(workspace_create: ResourceInCreate, workspace_repo: WorkspaceRepository = Depends(get_repository(WorkspaceRepository))) -> ResourceInResponse:
+    workspace = {}
+
     try:
         workspace = workspace_repo.create_workspace(workspace_create)
-        return ResourceInResponse(resource=workspace)
     except UnableToAccessDatabase:
         raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail=strings.STATE_STORE_ENDPOINT_NOT_RESPONDING)
 
+    try:
+        service_bus = ServiceBus()
+        await service_bus.send_resource_request_message(str(workspace.dict()))
```

Comment:
any particular reason why a dict is converted to str before passing to another class?

Reply:
The ServiceBusMessage constructor likes to ingest strings.